.pipelines/templates/deploy-helm-chart.yaml

```diff
@@ -0,0 +1,18 @@
+steps: 
```

Comment:
I think this should be renamed to `publish-helm-chart.yaml`. `deploy` makes me think this is just going to install the chart in the cluster